cli/tests/integration_tests.rs

```diff
@@ -2413,11 +2413,13 @@ mod integration {
       output: "test/test_finally_cleartimeout.out",
     });
 
-    itest!(unresolved_promise {
-      args: "test test/test_unresolved_promise.js",
-      exit_code: 1,
-      output: "test/deno_test_unresolved_promise.out",
-    });
+    // TODO(caspervonb) disabled a brief minute; how do we do this in a script execution context?
```

Comment:
Blocker

Reply:
Punting this, needs additional work.

Reply:
This should be reenabled before landing this PR, what's the problem here?

Reply:
Think we went over this in DMs last month or so but execute doesn't track unresolved promises.
So to handle this case we generate a tiny module that just calls `await runTests`.